playwright.config.js

```diff
@@ -29,6 +29,7 @@ module.exports = defineConfig({
 
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
     trace: 'on-first-retry',
+    testIdAttribute: 'data-test',
```

Comment:
great job!
let's add baseUrl into the config as well

Reply:
Done